<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>1025</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #100f10;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Screen" name="screen">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>850</width>
      <height>851</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: #ffa631;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sizeDown">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>890</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 20pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚ñ¶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sizeUp">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>890</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚ñ¶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delay">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>890</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>üïê</string>
    </property>
   </widget>
   <widget class="QPushButton" name="window">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 40pt ;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚ñ≠</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clip">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>800</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 40pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚úÇ</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>890</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 20pt &quot;Fira Code&quot;;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚èª</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polygon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚¨†</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleDown">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="autoRepeatDelay">
     <number>20</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="clock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚Üª</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="down">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚Üì</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pivot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚óâ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 25pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚ï±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeColor">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>890</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Emoji</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 20pt  &quot;Noto Emoji&quot;;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>üé®</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoRepeatDelay">
     <number>20</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚Üí</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="repaint">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>100</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 25pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>üñå</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="anticlock">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>600</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚Ü∫</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚Üë</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="reflect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>100</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 20pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>&lt;‚îä&gt;</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="shear">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>700</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 40pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚ñ±</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="location">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Fira Code&quot;;
qproperty-alignment: &quot;AlignCenter&quot;;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
</string>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt;
color: #ffa631;
border-width: 2px;
border-style: solid;
border-color: #cbc2c4;
background-color: #080808 ;
}

QPushButton:pressed {
color: #080808;
background-color: #ffa631;
}</string>
    </property>
    <property name="text">
     <string>‚Üê</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Screen</class>
   <extends>QGraphicsView</extends>
   <header location="global">screen.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
